src/lib/mongoBackend/mongoDiscoverContextAvailability.cpp

```diff
@@ -74,7 +74,8 @@ static HttpStatusCode processDiscoverContextAvailability
                           &count))
   {
     responseP->errorCode.fill(SccReceiverInternalError, err);
-    alarmMgr.dbError(responseP->errorCode.details);
+    // FIXME P10: overloggin (registrationQuery() false return menas that collectionRangedQuery() retursn false, corresponding to internal alarms in that function)
```

Comment:
If the error is already logged (and **always** logged in case it occurs), then this alarm is not necessary.
The question is always, which one should be removed, low level or high level?

I guess the best info is given at lower level in this case, so

'LGTM'